movies/src/__tests__/app.spec.js

```diff
@@ -0,0 +1,36 @@
+const request = require('supertest');
```

Comment:
Can we have it in a `test` directory rather than in `__test__`?

Reply:
oh c'mon, why do you care? ;)

Reply:
It's a common convention and it prevents the test dir from being visually lost within the "real" dirs.

Reply:
Hmm - I'm not convinced it is a common approach. There are many alternatives, i.e. e2e/tests/src approach. AFAIK this is common for the framework used - I'd like to have framework agnostic approach.

Reply:
Mosts frameworks don't care where you place the tests and how you name them, or are at least configurable in that regard. I renamed it to "tests" now.